Process.h

```diff
@@ -21,6 +22,13 @@ in the source distribution for its full text.
 #define SYS_ioprio_set __NR_ioprio_set
 #endif
 
+// On Linux, this works only with glibc 2.1+. On earlier versions
```

Comment:
PAGE_SIZE has been refactored, see https://github.com/htop-dev/htop/commit/361877454fe56c95a995d5bdbb7eb70c21e39d62

Reply:
Oops, probably an oversight while rebasing the patches …